recipes/magit

```diff
@@ -1,26 +1,8 @@
 (magit :fetcher github
        :repo "magit/magit"
-       :files ("lisp/magit-utils.el"
-               "lisp/magit-section.el"
-               "lisp/magit-git.el"
-               "lisp/magit-mode.el"
-               "lisp/magit-process.el"
-               "lisp/magit-core.el"
-               "lisp/magit-diff.el"
-               "lisp/magit-wip.el"
-               "lisp/magit-apply.el"
-               "lisp/magit-log.el"
-               "lisp/magit.el"
-               "lisp/magit-sequence.el"
-               "lisp/magit-commit.el"
-               "lisp/magit-remote.el"
-               "lisp/magit-bisect.el"
-               "lisp/magit-stash.el"
-               "lisp/magit-blame.el"
-               "lisp/magit-ediff.el"
-               "lisp/magit-extras.el"
-               "lisp/magit-autorevert.el"
+       :files ("lisp/magit-*.el"
```

Comment:
@syohex @purcell I noticed that `magit.el` is not included here. Is it OK?
before: 
https://travis-ci.org/milkypostman/melpa/builds/101990888
after: 
https://travis-ci.org/milkypostman/melpa/builds/102039195